packages/automerge-doc-server/src/postgres_storage_adapter.ts

```diff
@@ -0,0 +1,75 @@
+import { Pool, QueryResult } from "pg";
+import {
+  StorageAdapterInterface,
+  type StorageKey,
+} from "@automerge/automerge-repo/slim"
+
+export class PostgresStorageAdapter implements StorageAdapterInterface {
+    private pool: Pool;
+
+    constructor(pool: Pool) {
+        this.pool = pool;
+    }
+
+    // Load the data for an exact key match
+    async load(key: StorageKey): Promise<Uint8Array | undefined> {
+        const result = await this.pool.query(
+            "SELECT data FROM storage WHERE key = $1",
```

Comment:
I figure there are database migrations that should go with this PR. Based on a quick look, we've got:

- a new table called `storage` with the columns `data` and `key`
- a new column in `refs` called `doc_id`

Reply:
yup, they are now added

"it works on my machine"... Ideally we'd be testing the migrations in CI, but given the infrequency of db changes I doubt that's worth it right now.